test/commands/test_parse.js

```diff
@@ -34,4 +62,84 @@ describe('parse', () => {
     assert(!fs.existsSync(path.resolve('../../mvnw/target')));
     done();
   });
+
+  it('rejects invalid parser version', function () {
+    this.timeout(30000);
+    const home = path.resolve('temp/test-parse/invalid-version');
+    createTestProject(home);
+
+    const result = runCommand([
+      'parse',
+      '--parser=999.999.999',
+      '-s', path.resolve(home, 'src'),
+      '-t', path.resolve(home, 'target'),
+    ].join(' '));
+
+    assert(result.status !== 0, 'Command should fail with non-zero exit code');
+  });
+
+  it('accepts valid parser version', function () {
+    this.timeout(60000);
+    const home = path.resolve('temp/test-parse/valid-version');
+    createTestProject(home);
+
+    const result = runCommand([
+      'parse',
+      '--parser=0.28.11',
+      '-s', path.resolve(home, 'src'),
+      '-t', path.resolve(home, 'target'),
+    ].join(' '));
+
```

Comment:
@vladislav-yevtushenko it's better not to leave empty lines inside function body: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html